lib/Product.php

```diff
@@ -33,6 +33,7 @@
  * @property null|\Stripe\StripeObject $package_dimensions The dimensions of this product for shipping purposes.
  * @property null|bool $shippable Whether this product is shipped (i.e., physical goods).
  * @property null|string $statement_descriptor Extra information about a product which will appear on your customer's credit card statement. In the case that multiple products are billed at once, the first statement descriptor will be used.
+ * @property null|string|\Stripe\TaxCode $tax_code A <a href="https://stripe.com/docs/tax/tax-codes">tax code</a> ID.
```

Comment:
This link is dead, is it going to be rolled out as part of this release?

Reply:
Yes. If you log in and your account is opted-in you'll be able to see it now.
ptal @seang-stripe